src/services/GraphQl/client.js

```diff
@@ -0,0 +1,41 @@
+
+import { split, HttpLink,ApolloClient, InMemoryCache  } from '@apollo/client';
+import { getMainDefinition } from '@apollo/client/utilities';
+import { WebSocketLink } from '@apollo/client/link/ws';
+
+
+
+const httpLink = new HttpLink({
+  uri: 'https://azordev-dasher-db.herokuapp.com/v1/graphql'
+});
+
+const wsLink = new WebSocketLink({
+  uri: 'ws://azordev-dasher-db.herokuapp.com/v1/graphql',
```

Comment:
This is causing an error on production, please check if changing it to `wss` works
![image](https://user-images.githubusercontent.com/36519478/135971990-c2782da1-a0c1-4166-b3b5-3e3b89adc9e7.png)
```suggestion
  uri: 'wss://azordev-dasher-db.herokuapp.com/v1/graphql',
```